src/cryptoadvance/specter/templates/settings/bitcoin_core_settings.jinja

```diff
@@ -31,7 +31,7 @@
 				Bitcoin Core data directory path:<br><input type="text" id="datadir" name="datadir" type="text" value="{{ datadir }}">
 			</span>
 			<br><br>
-			<div id="rpc_settings" style="padding: 10px; border-radius: 10px;">
```

Comment:
Removed the padding to be consistent with the rest of the bitcoin core settings form, incl. the other setting pages. Border radius not needed, since no border exists for the rpc_settings div.